src/pds_doi_service/core/actions/test/update_test.py

```diff
@@ -331,6 +331,11 @@ def test_invalid_update_requests(self):
         with self.assertRaises(CriticalDOIException):
             self._update_action.run(**update_kwargs)
 
+    #@patch.object(
+    #
+    #)
+    #def test_update_doi(self):
+    #    pass
```

Comment:
Two simple rules for dealing with commented-out code:

1. Always add an explanation telling _why_ it's commented out so people in the future won't wonder if it was needed or the circumstances for adding it back
2. If rule ⑴ cannot be applied, then _delete_ the code immediately, so people in the future won't _have to wonder_ if it was actually needed 😇